site/content/en/docs/Guides/Client SDKs/csharp.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "C# Game Server Client SDK"
+linkTitle: "C#"
+date: 2019-06-13T10:17:50Z

```

Comment:
dates seem off? ;)

Reply:
You caught my lazy copy-pasting! 😅 Done! :)